Maestro/subscriptions.json

```diff
@@ -299,6 +305,13 @@
         }
       }
     },
+    // Mirror corefx changes into VSO
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/**/*",
```

Comment:
This points to changes in the versions repo, and should point to the corefx repo instead: `https://github.com/dotnet/corefx/blob/master/**/*`